I couldn't get *.cmxs files to build in the cross-compiler.  This
patch disables them.

diff --exclude _build -urN ocaml-3.11.0+beta1.orig/otherlibs/Makefile.shared ocaml-3.11.0+beta1.mingw/otherlibs/Makefile.shared
--- ocaml-3.11.0+beta1.orig/otherlibs/Makefile.shared	2008-07-15 16:31:32.000000000 +0100
+++ ocaml-3.11.0+beta1.mingw/otherlibs/Makefile.shared	2008-11-15 15:31:48.000000000 +0000
@@ -42,7 +42,8 @@
 
 all: lib$(CLIBNAME).$(A) $(LIBNAME).cma $(CMIFILES)
 
-allopt: lib$(CLIBNAME).$(A) $(LIBNAME).cmxa $(LIBNAME).$(CMXS) $(CMIFILES)
+allopt: lib$(CLIBNAME).$(A) $(LIBNAME).cmxa $(CMIFILES)
+# $(LIBNAME).$(CMXS)
 
 $(LIBNAME).cma: $(CAMLOBJS)
 	$(MKLIB) -o $(LIBNAME) -oc $(CLIBNAME) -ocamlc '$(CAMLC)' -linkall $(CAMLOBJS) $(LINKOPTS)
@@ -50,8 +51,8 @@
 $(LIBNAME).cmxa: $(CAMLOBJS_NAT)
 	$(MKLIB) -o $(LIBNAME) -oc $(CLIBNAME) -ocamlopt '$(CAMLOPT)' -linkall $(CAMLOBJS_NAT) $(LINKOPTS)
 
-$(LIBNAME).cmxs: $(LIBNAME).cmxa lib$(CLIBNAME).$(A)
-	$(CAMLOPT) -shared -o $(LIBNAME).cmxs -I . $(LIBNAME).cmxa
+#$(LIBNAME).cmxs: $(LIBNAME).cmxa lib$(CLIBNAME).$(A)
+#	$(CAMLOPT) -shared -o $(LIBNAME).cmxs -I . $(LIBNAME).cmxa
 
 lib$(CLIBNAME).$(A): $(COBJS)
 	$(MKLIB) -oc $(CLIBNAME) $(COBJS) $(LDOPTS)
